recipe/meta.yaml

```
@@ -14,7 +14,7 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
```

I don't think you can put quotes around the hash type. I think the yaml reader has problems with this
```suggestion
  {{ hash_type }}: '{{ hash_value }}'
```